package.json

```diff
@@ -5,7 +5,8 @@
   "main": "run.js",
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
```

Comment:
Should this be updated to : "build": "tsc --outDir ./lib --rootDir ./src", ?

Reply:
Its mentioned in tsconfig so does not need to be.